### DISCUSSION TITLE
Building subfinder

### DISCUSSION THREAD

**Discussion by ghost** [projectdiscovery/subfinder#1070] [Q&A]
Just a simple go build produces a binary with 37.7MB of size
Using go build -ldflags "-w" reduces the binary to 30.7MB
The release file subfinder_2.6.3_linux_amd64.zip meanwhile have the size of 25.9MB
How did you manage to make the binary size smaller. Would you mind sharing your build instructions. Thank you.

**Comment by MetzinAround** [comment]
I think it's smaller since it's a zip file, but I'll defer to @ehsandeep on how it was built!
